specification/sql/resource-manager/Microsoft.Sql/preview/2022-02-01-preview/RecoverableDatabases.json

```diff
@@ -0,0 +1,193 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-02-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/recoverableDatabases": {
+      "get": {
+        "tags": [
+          "RecoverableDatabases"
+        ],
+        "description": "Gets a list of recoverable databases.",
+        "operationId": "RecoverableDatabases_ListByServer",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of recoverable databases.",
+            "schema": {
+              "$ref": "#/definitions/RecoverableDatabaseListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get list of recoverable databases.": {
+            "$ref": "./examples/RecoverableDatabaseList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/recoverableDatabases/{recoverableDatabaseName}": {
+      "get": {
+        "tags": [
+          "RecoverableDatabases"
+        ],
+        "description": "Gets a recoverable database.",
+        "operationId": "RecoverableDatabases_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "recoverableDatabaseName",
```

Comment:
RecoverableDatabases_Get in 2014-04-01-stable, it is `databaseName`. 
however, to generate the swagger, we have to change to `recoverableDatabaseName`

Reply:
update the parameter to the legacy API contract